packages/statemanager/src/cache.ts

```diff
@@ -1,8 +1,10 @@
 import { Account } from '@ethereumjs/util'
+// eslint-disable-next-line implicit-dependencies/no-implicit
+import { OrderedMap } from 'js-sdsl'
 
 import type { Address } from '@ethereumjs/util'
-
-const Tree = require('functional-red-black-tree')
+// eslint-disable-next-line implicit-dependencies/no-implicit
```

Comment:
Hmm, these shouldn't be necessary I guess, maybe you've forgotten to rebuild or to restart your editor or something? 🤔

Anyhow: won't trigger a new CI run just for that, have added to our #2105 issue for small fixes.